### PR TITLE
Add manual real-world QA instructions

### PR DESCRIPTION
**Pull request by amyrlam**
## :pushpin: Summary

Resolves: part of https://github.com/hashicorp/flight/issues/240

One more tweak to the instructions, seems like following this PR, @MelSumner can give https://github.com/hashicorp/flight/issues/240 a try at any point that's convenient this week.

### CODE REVIEW

File: ember-flight-icons/CONTRIBUTING.md
Comment (MelSumner) [inline ember-flight-icons/CONTRIBUTING.md:57]: Question: Are you sure we need this? We already have two ways to test that the icons render: 

1. we have a test that explicitly checks it
2. you could run the dummy app if you wanted to see it work manually

We have tests that make sure the icons render- also running the dummy app would be a manual way to test that the icons render.

Comment (amyrlam) [reply ember-flight-icons/CONTRIBUTING.md:57]: Happy to remove. Was more adding this as a placeholder, could be `cloud-ui` in the future.

These changes https://github.com/hashicorp/flight/pull/245 worked in the dummy app but not `cloud-ui`, maybe cause of the nested `packages/`, not sure yet.

Comment (amyrlam) [reply ember-flight-icons/CONTRIBUTING.md:57]: cloud-ui PR: https://github.com/hashicorp/cloud-ui/pull/1322 but did a `yarn link` to GitHub branch

Comment (MelSumner) [reply ember-flight-icons/CONTRIBUTING.md:57]: I might be mis-understanding the purpose, let me read through it again and we can sync tomorrow 👍

Comment (didoo) [reply ember-flight-icons/CONTRIBUTING.md:57]: I think we can leave it for now, is a way to QA the icons "in production" (I understand we have a dummy app, but that's a more "controlled" environment, while we're not sure what happens in the real apps).
Later, when the process of releasing icons is consolidated, and we feel more confident that we're not breaking anything, we can definitely remove it.

@amyrlam I would suggest to add a paragraph/intro at the beginning of the two sections in which we explain what we're trying to do (if I am reading the CONTRIBUTING doc, I have no idea why I jump to a completely different project/repository and what that repo is for. (and maybe add is a temporary solution)

**Review by didoo — APPROVED** [review]
For me is OK, so I'll approve. @amyrlam maybe wait for @MelSumner re-reading before merging, so we're all on the same page.

File: ember-flight-icons/CONTRIBUTING.md
Comment (amyrlam) [reply ember-flight-icons/CONTRIBUTING.md:57]: @MelSumner @didoo i've updated the instructions and changed to `cloud-ui`. i'll merge in when CI is green. melanie can definitely make further edits when she completes https://github.com/hashicorp/flight/issues/240, it's a living document

**Pull request merged**
